ZenPacks/zenoss/PythonCollector/zenpython.py

```diff
@@ -332,6 +343,20 @@ def __init__(self, taskName, configId, scheduleIntervalSeconds, taskConfig):
         self.metricExtraTags = getattr(
             self._dataService, "metricExtraTags", False)
 
+    def getDatasources(self):
+        try:
+            (template, datasource) = self.choosenDatasource.split('/')
```

Comment:
Parenthesis around destructured variable names aren't required.
```suggestion
            template, datasource = self.choosenDatasource.split('/')
```